portal-kernel/src/com/liferay/portal/kernel/search/dummy/DummyIndexer.java

```diff
@@ -137,17 +137,6 @@ public void postProcessContextBooleanFilter(
 		throws Exception {
 	}
 
-	/**
```

Comment:
@dantewang @yuhai ,
There is still on usage of AssetEntriesFacet:
`modules/apps/portal-rules-engine/portal-rules-engine-sample-web/src/main/resources/dependencies/rules_user_custom_attribute_content.drl:	Facet assetEntriesFacet = new AssetEntriesFacet(searchContext);
`